custom_components/london_tfl/sensor.py

```diff
@@ -206,3 +210,9 @@ def extra_state_attributes(self):
         attributes['data'] = data
 
         return attributes
+
+    def shortenPlatformName(destinationName, shorten):
+        if CONF_SHORTEN_STATION_NAMES:
+            return destinationName.replace("Underground Station", "").trim()

```

Comment:
Could generalise by having a list (in const.py) of strings to remove, and iterating over them here.